### PR TITLE
ARTEMIS-3374 config managed queue can be deleted by durable subscriber

### PR DESCRIPTION
**Pull request by jbertram**

### CODE REVIEW

File: tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/JMSMessageConsumerTest.java [hunk 1]
```diff
@@ -217,6 +220,30 @@ private void doTestSelector(boolean topic) throws Exception {
       }
    }
 
+   @Test(timeout = 60000)
+   public void testDurableSubscriptionWithConfigurationManagedQueue() throws Exception {
```
Comment (michaelandrepearce) [inline tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/JMSMessageConsumerTest.java:224]: Should maybe this same test be extended and checked for openwire and core

**Comment by michaelandrepearce** [comment]
Should ensure same behaviour on core and openwire.

**Comment by jbertram** [comment]
@michaelandrepearce, good call. I updated the test and the fix.

**Review by michaelandrepearce — APPROVED** [review]
Lgtm. Nice stuff @jbertram as always

**Pull request closed without merging**